pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -364,15 +364,15 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 		},
 		{
 			Type:        "victorops",
-			Name:        "VictorOps",
-			Description: "Sends notifications to VictorOps",
-			Heading:     "VictorOps settings",
+			Name:        "Splunk OnCall",
```

Comment:
Please don't revert the changes here though as `ngalert` is the new Grafana Alerting.